state-chain/pallets/cf-swapping/src/migrations/swap_and_swap_request_migration.rs

```diff
@@ -89,57 +89,6 @@ impl<T: Config> UncheckedOnRuntimeUpgrade for Migration<T> {
 	}
 
 	fn on_runtime_upgrade() -> Weight {
-		crate::SwapRequests::<T>::translate_values::<old::SwapRequest<T>, _>(|old_swap_requests| {
```

Comment:
We need to branch 1.8 and remove this migration (unless we want this PR in 1.8, in which case I can fix this migration properly).